### ISSUE TITLE
Import event data from external source

### ISSUE THREAD

**Issue by pangelo** [lcdporto/events-clock#7]
Currently, the event list for the clock needs to be created from scratch for each instance of the app.

Instead of creating the events on the list one by one, it would be nice to pull/select events from an external source like:
- an online calendar (e.g a CalDAV endpoint)
- an event webpage with embedded hCalendar microformat
- a specific JSON or XML file

This would work by having a text entry field for a URL and a button labelled "Import events" in the /set-events page, in which you could input the URL of the external data source (and maybe  also have a dropdown to specify the format, although this might be inferred from the URL).

Upon pressing the button the app would fetch the external resource (we might need to set CORS headers on the server to allow this) parse it an add the events found to the event list.

**Comment by ighormartins** [comment]
Since #12, it's possible to share a link that will load event data into the local storage.
I'm gonna leave this open as having other types of integrations would be nice too.

**Comment by pangelo** [comment]
Yeah, sharing  a view URL solves the problem of remote synchronization for users. I changed the wording on the issue to reflect that this might still be useful for those creating the event list in the first place.
